### DISCUSSION TITLE
Grafana and tinydb

### DISCUSSION THREAD

**Discussion by pkp124** [msiemens/tinydb#556] [General]
Hello,

I am new to tinydb and wanted to learn more about the integrations for tinydb. I would like to use tinydb to create a dashboard for benchmarking. I was curious what visualization tools can be used. If grafana can be used, that would be brilliant. Any suggestions are welcome.

Thanks

**Comment by msiemens** [comment]
As far as I know, there is no Grafana datasource for TinyDB, nor any other form of integration
